app/views/submissions/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= simple_form_for @submission  do |form| %>
```

Comment:
As a general rule of thumb, it's good practice to avoid using instance variables in partials. That couples them to being rendered from views whose controllers defined the instance variable.

Instead, use local variables. That way the partial can be rendered from anywhere as long as you pass the correct locals.